src/chrome/content/rules/Pingupod.de.xml

```diff
@@ -3,7 +3,6 @@
 	<!--	Direct rewrites:
 				-->
 	<target host="pingupod.de" />
-	<target host="www.pingupod.de" />
```

Comment:
@J0WI  added the documentation but didn't manage to get the redirects pass the tests.
I tried something like:

	<test url="http://www.pingupod.de/" />
	<test url="http://mgp.pingupod.de/" />
	<test url="http://diaspora.pingupod.de/" />

	<rule from="^http://(www|mgp|diaspora)\.pingupod\.de/"
		to="https://pingupod.de/" />

and got:

ERROR src/chrome/content/rules/Pingupod.de.xml: Not enough tests (0 vs 3) for <Rule from '^http://(www|mgp|diaspora)\.pingupod\.de/' to 'https://pingupod.de/'>

am I doing something wrong?